### PR TITLE
Added classroom counter to dashboard home

### PR DESCRIPTION
**Pull request by usmanmuhd**
Updated screenshot:
screenshot from 2019-02-24 21-20-01

### CODE REVIEW

File: dashboard/views/dashboard.ejs [hunk 1]
```diff
@@ -54,6 +54,22 @@
 							</div>
 						</div>
 					</div>
+					<div class="col-lg-4 col-md-4 col-sm-4">
```
Comment (tarunsinghal92) [inline dashboard/views/dashboard.ejs]: change the column width to `col-lg-3 col-md-3 col-sm-3` so that they appear in a line.

Comment (tarunsinghal92) [reply dashboard/views/dashboard.ejs]: do it for all 4 cards

File: dashboard/views/dashboard.ejs [hunk 2]
```diff
@@ -54,6 +54,22 @@
...
+						<div class="card card-stats">
+							<div class="card-header" data-background-color="black">
+								<p class="custom-icon activity-icon"></p>
```
Comment (tarunsinghal92) [inline dashboard/views/dashboard.ejs]: change classroom icon on the card; use the one sidebar as reference.

**Comment by tarunsinghal92** [comment]
@usmanmuhd Please check the comments.

**Comment by usmanmuhd** [comment]
@tarunsinghal92 Updated the PR. Please take a look.

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,8 @@
 node_modules
 dashboard/node_modules
 api/test/fakestats.js
+
+# database and docker files
+docker/Dockerfile-mongodb
```
Comment (tarunsinghal92) [inline .gitignore]: @usmanmuhd 

what are these added in .gitignore?

Comment (usmanmuhd) [reply .gitignore]: @tarunsinghal92 The first 2 are the files generated by `generate-docker-compose.sh` and the last one is the mongo container data. They both are not needed to be uploaded by anyone. Thought it's better to add it to .gitignore.

Comment (tarunsinghal92) [reply .gitignore]: okay!

**Pull request merged**